packages/nx/src/config/workspaces.ts

```diff
@@ -100,6 +100,24 @@ export class Workspaces {
     return this.cachedWorkspaceConfig;
   }
 
+  private mergeWorkspaceJsonAndGlobProjects(
+    workspaceJsonProjects: { [name: string]: any },
+    globProjects: { [name: string]: any }
+  ) {
+    const res = workspaceJsonProjects;
+    const folders = {};
```

Comment:
Should we use a `Set` object for this rather than the object since it is just a Record<string, bool>?